app/util/bitbucket/populate_db.sh

```diff
@@ -20,7 +20,7 @@ BITBUCKET_DB_USER="postgres"
 BITBUCKET_DB_PASS="Password1!"
 
 # BITBUCKET version variables
-SUPPORTED_BITBUCKET_VERSIONS=(6.10.5 7.0.5 7.6.0)
+SUPPORTED_BITBUCKET_VERSIONS=(7.6.0 7.0.5 6.10.5)
```

Comment:
do not change the order here, this order is used in by `force` flag to identify the latest version.